src/views/Configure.vue

```diff
@@ -12,6 +12,7 @@
         <file-system
           :roots="['config']"
           :max-height="816"
+          :hideable-files-available="true"
```

Comment:
What's the use case for this? Seems redundant if it's hardcoded to true..

Reply:
It is to avoid unnecessary filtering in other folders. But I could change it on a simple check with the root (=== 'config')

Reply:
Ahh, I see. I'm fine with either, I'll let @pedrolamas decide

Reply:
That is now gone, so should not be a problem anymore!